### PR TITLE
#1-4Task. Dzmitry Bryt

### PR DESCRIPTION
**Pull request by Daqueze21**
https://travis-ci.org/github/Daqueze21/js-assignments

### CODE REVIEW

File: task/04-arrays-tasks.js [hunk 1]
```diff
@@ -9,257 +9,254 @@
...
- * 
+ * @param {number} n
+ *
  * @example
  *    [ 1, 3, 4, 5 ], 2  => [ 4, 5 ]
  *    [ 'a', 'b', 'c', 'd'], 3  => [ 'b', 'c', 'd' ]
  */
 function getTail(arr, n) {
-   throw new Error('Not implemented');
+  return arr.slice(arr.length-n) ;
```
Comment (FedorovD) [inline task/04-arrays-tasks.js:173]: should return n last items in the array

**Comment by FedorovD** [comment]
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

File: task/04-arrays-tasks.js (hunk 1 above)
Comment (Daqueze21) [reply task/04-arrays-tasks.js:173]: is my solution wrong?

Comment (Daqueze21) [reply task/04-arrays-tasks.js:173]: > should return n last items in the array

is my solution wrong?

Comment (Daqueze21) [reply task/04-arrays-tasks.js:173]: > should return n last items in the array

i need to add another solution and new pull PR?

**Comment by FedorovD** [comment]
Closing the PR as accepted! Thanks!

**Pull request closed without merging**